pallets/proofs-dealer/src/lib.rs

```diff
@@ -134,15 +131,29 @@ pub mod pallet {
         /// `PriorityChallengesQueue` in the `BlockToChallenges` StorageMap. These checkpoint challenge
         /// rounds have to be answered by ALL Providers, and this is enforced by the `submit_proof`
         /// extrinsic.
+        ///
+        /// WARNING: This period needs to be equal or larger than the challenge period of the smallest
```

Comment:
Yeah, that would be a good way to prevent runtime developers from making a mistake with this. My long term goal is to generalise the `submit_proof` extrinsic to check for an arbitrary number of checkpoint challenges in between the last two challenges of a given provider. Therefore removing this limitation.

Reply:
Will add an integrity test for this in another PR